driver/pace/driver/run.py

```diff
@@ -116,82 +130,179 @@ class SerialboxConfig(InitializationConfig):
     """
 
     path: str
-    npx: int
-    npy: int
-    npz: int
-    layout: List[int]
+    serialized_grid: bool
     backend: str
+    rebuild: bool
+    validate_args: bool
+    format_source: bool
+    device_sync: bool
 
     @property
     def start_time(self) -> datetime:
         return datetime(2000, 1, 1)
 
+    @property
+    def f90_namelist(self) -> f90nml.Namelist:
+        return f90nml.read(self.path + "/input.nml")
+
+    @property
+    def namelist(self) -> Namelist:
+        return Namelist.from_f90nml(self.f90_namelist)
+
+    @property
+    def stencil_config(self) -> pace.dsl.stencil.StencilConfig:
+        return pace.dsl.stencil.StencilConfig(
+            backend=self.backend,
+            rebuild=self.rebuild,
+            validate_args=self.validate_args,
+        )
+
+    def get_serialized_grid_damping_coeff_and_driver_grid(
```

Comment:
Can these be private methods/attributes? They should only be called from inside this class.

Reply:
Changed the last two. The namelists are actually called in `_set_kwargs_from_serialbox` inside `DriverConfig`. We override the options to be from fortran namelist there.